Jellyfin.Networking/Configuration/NetworkConfiguration.cs

```diff
@@ -227,5 +227,10 @@ public string BaseUrl
         /// Gets or sets the known proxies.
         /// </summary>
         public string[] KnownProxies { get; set; } = Array.Empty<string>();
+
+        /// <summary>
+        /// Gets or sets the known networks.
+        /// </summary>
+        public string[] KnownNetworks { get; set; } = Array.Empty<string>();
```

Comment:
This is no longer backport material tbh

Reply:
Only included it as a BP as the known proxies was added in 10.7 but not this secondary function - which a user has already highlighted is needed.

Happy for it be BP or accepted on next version.

Reply:
Needed for what?

Reply:
It's along the same lines as KnownProxies. 

_"Address ranges of known networks to accept forwarded headers from."_

_"The request's original remote IP must match an entry in the KnownProxies or KnownNetworks lists before forwarded headers are processed. This limits header spoofing by not accepting forwarders from untrusted proxies. When an unknown proxy is detected, logging indicates the address of the proxy:"_

https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer?view=aspnetcore-5.0&viewFallbackFrom=aspnetcore-5.0%20.

Reply:
I know what it's for, but what's the use case for our users? Is it common for a reverse proxy to have a dynamic IP? Doesn't seem like something super important to me.

Reply:
@cvium: see matrix

Reply:
@cvium not super important? ever used k8s or swarm? you can't simply set static ip there. It's not like it is rocket science to add this simple functionality. I got this working with basically adding two lines of code.

Reply:
See https://github.com/BaronGreenback/jellyfin/pull/118

Reply:
I don't use k8s. It's overkill for what I'm doing, but I'll take your word for it.

Reply:
Just to chime in, I haven't been following this in detail but - I'm not using K8S but this is indeed important functionality for me. My proxies (multiple) obtain IPs from DHCP, the only static part is their VIP (shared between them). Thus it's impossible for me to know at any given moment what the real IPs of the proxies are.

Thus my original ask that I believe prompted all this: the ability to set something other than a static list of IPs, either a whole netblock, or just turning off the functionality, if required, so that I can tell JF to trust X-Forwarded-For from my proxies without ever reconfiguring them.

Reply:
@joshuaboniface The code takes individual proxies lists, trusted networks, and hosts names if that is the way in which the user wants to go.

Alternatively it can be left all blank.